keystore/src/entities/mod.rs

```diff
@@ -114,16 +114,16 @@ impl EntityFindParams {
     pub fn to_sql(&self) -> String {
         use std::fmt::Write as _;
         let mut query: String = "".into();
-        if let Some(limit) = self.limit {
-            let _ = write!(query, " LIMIT {limit}");
-        }
         if let Some(offset) = self.offset {
             let _ = write!(query, " OFFSET {offset}");
         }
         let _ = write!(query, " ORDER BY rowid");
         if self.reverse {
             let _ = write!(query, " DESC");
         }
+        if let Some(limit) = self.limit {
+            let _ = write!(query, " LIMIT {limit}");
+        }
```

Comment:
The message of this commit says
```
 fix: EntityFindParams SQL keyword ordering 
 ```
but to me it isn't obvious what the problem was. It would be good to add another line to the commit message. :sweat_smile: